stan/math/opencl/kernel_generator/optional_broadcast.hpp

```diff
@@ -53,10 +53,9 @@ class optional_broadcast_
 
   /**
    * generates kernel code for this and nested expressions.
-   * @param[in,out] generated set of already generated operations
-   * @param ng name generator for this kernel
    * @param row_idx_name  row index variable name
    * @param col_idx_name  column index variable name
+   * @param var_name_arg name generator for this kernel
```

Comment:
This one is probably on me. But I think this should be caught on tests on the PR. 

Anyway this should be changed to:
```suggestion
   * @param var_name_arg name of the variable in kernel that holds argument to
   * this expression
```